Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -41,9 +41,12 @@
 //
 // Limit Switches
 //
-#define X_STOP_PIN         35
-#define Y_STOP_PIN         12
-#define Z_STOP_PIN         36
+#define X_MIN_PIN          35
+#define X_MAX_PIN          -1
+#define Y_MIN_PIN          -1
+#define Y_MAX_PIN          12
+#define Z_MIN_PIN          36
+#define Z_MAX_PIN          -1
```

Comment:
Does this change fix something for you? The idea here is, if there are only 3 plugs on the board, then assign them to the "STOP" pins. They will be re-assigned to the "MIN/MAX" pins according to the configuration. So this change actually breaks intended behavior.